packages/ui/src/components/shadcn/ui/badge.tsx

```diff
@@ -3,34 +3,32 @@ import * as React from 'react'
 
 import { cn } from '../../../lib/utils/cn'
 
-const badgeVariants = cva(
-  'inline-flex items-center px-2.5 py-0.5 rounded-full text-xs bg-opacity-10',
-  {
-    variants: {
-      variant: {
-        default: 'bg-surface-200 text-foreground-light border border-strong',
-        warning: 'bg-warning text-warning border border-warning-500',
-        success: 'bg-brand text-brand-600 border border-brand-500',
-        destructive: 'bg-destructive text-destructive-600 border border-destructive-500',
-        brand: 'bg-brand text-brand-600 border border-brand-500',
-        secondary:
-          'bg-secondary hover:bg-secondary/80 border-transparent text-secondary-foreground',
-        outline: 'bg-transparent text border border-foreground-muted',
-      },
-      size: {
-        small: 'px-2.5 py-0.5 text-xs',
-        large: 'px-3 py-0.5 rounded-full text-sm',
-      },
-      dot: {
-        true: '-ml-0.5 mr-1.5 h-2 w-2 rounded-full',
-      },
+const badgeVariants = cva('inline-flex items-center rounded-full font-normal', {
+  variants: {
+    variant: {
+      default: 'bg-surface-75 text-foreground-light border border-strong',
+      warning: 'bg-warning bg-opacity-10 text-warning border border-warning-500',
+      success: 'bg-brand bg-opacity-10 text-brand-600 border border-brand-500',
+      destructive:
+        'bg-destructive bg-opacity-10 text-destructive-600 border border-destructive-500',
+      brand: 'bg-brand bg-opacity-10 text-brand-600 border border-brand-500',
+      secondary:
+        'bg-secondary bg-opacity-10 hover:bg-secondary/80 border-transparent text-secondary-foreground',
+      outline: 'bg-transparent text border border-foreground-muted',
     },
-    defaultVariants: {
-      variant: 'default',
-      size: 'small',
+    size: {
+      small: 'px-2 py-0.5 text-xs',
```

Comment:
This is the only visual change: `small` has a `px-2` instead of `px-2.5`. Everything else here is a style cleanup. A lot of duplicity.